src/main/resources/locales/cy/translation/sex-and-title.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
This file is added in another PR (https://github.com/hmcts/et-sya-frontend/pull/389) with translations so may be this can be removed from here?

Reply:
This should be fine when we merge the PRs, we can just push your filled in keys over this file